### PR TITLE
Format phone numbers with +91 at the beginning

### PR DESCRIPTION
**Pull request by dmydlo**
Hi @calellowitz,
I have changed the formatting of phone numbers in the tabular reports to start with +91.
Need QA: Yes
Environment: n/a
Locally Tested: Yes
Regards, Dawid.

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/pregnant_women.py [hunk 1]
```diff
@@ -14,6 +14,9 @@ class PregnantWomenExport(ExportableMixin, SqlData):
     title = 'Pregnant Women'
     table_name = 'agg_ccs_record_monthly'
 
+    def phone_number_fucntion(self, x):
```
Comment (calellowitz) [inline custom/icds_reports/sqldata/exports/pregnant_women.py]: spelling (same on all of them)

**Review by calellowitz — CHANGES_REQUESTED** [review]
Is there a way this could be pulled into a separate util function instead of being defined on all of them.

**Review by calellowitz — APPROVED** [review]

File: custom/icds_reports/utils/__init__.py [hunk 2]
```diff
@@ -1142,3 +1142,7 @@ def get_datatables_ordering_info(request):
     order_by_name_column = request.GET.get('columns[%s][data]' % order_by_number_column)
     order_dir = request.GET.get('order[0][dir]', 'asc')
     return start, length, order_by_number_column, order_by_name_column, order_dir
+
+
+def phone_number_function(x):
+    return "+{0}{1}".format('' if str(x).startswith('91') else '91', x) if x else x
```
Comment (kaapstorm) [inline custom/icds_reports/utils/__init__.py]: We already use the [phonenumberslite][1] library. You could do this more generically with
```python
    def phone_number_function(x, from_country='IN'):
        number = phonenumbers.parse(x, from_country)
        return phonenumbers.format_number(number, phonenumbers.PhoneNumberFormat.INTERNATIONAL)
```

  [1]: https://github.com/daviddrysdale/python-phonenumbers

Comment (kaapstorm) [reply custom/icds_reports/utils/__init__.py]: (That's not to say that you **should**. Use it / don't use it.)

**Comment by calellowitz** [comment]
Merging for now, to get this out, but @kaapstorm's comments would be a good followup @dmydlo

**Pull request merged**